kubernetes/config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: issecrets
+type: Opaque
+data:
+  log-level:
```

Comment:
All of these non-sensitive parameters should not be in the secret. These are more appropriate for a configmap.